doc/architecture/decisions/0002-git-commit-messages.md

```diff
@@ -0,0 +1,45 @@
+# 2. git commit messages
+
+Date: 2019-11-13
+
+## Status
+
+Draft
+
+## Context
+
+We need to improve the quality of the git commit messages so developers can easily understand the context of the proposed changes, throughout a concise and consistent message in order to avoid lengthy code reviews and project long term maintainability.
```

Comment:
This and some other lines below are too long. Please, use a max length of 100 characters.

Reply:
Unnecessary comma before `throughout`.

`... code reviews and project long term maintainability.` -> `code reviews and long-term project maintainability.`

Reply:
is the line length a project specification? I am asking 'cause nowadays 120 chars is a solid number...

Reply:
120 characters in code is… mostly fine (it breaks side-by-side diffs; in _this view_ I have this line wrapped into 3, the longest is ` context of the proposed changes… to avoid length` at 102 characters).

Also, this line is 252 characters long.

Reply:
I agree with @mook-as.